### PR TITLE
Fix pytest command line in hatch test scripts

### PR DESCRIPTION
**Pull request by mcarans**
This fixes the tests failing to run in GitHub Actions. Does it looks ok to you @turnerm ?

What I don't get is that when I run locally `hatch run test:test`, it fails with:
```
tests/hdx/conftest.py:7: in <module>
    from hdx.utilities.downloader import Download
E   ModuleNotFoundError: No module named 'hdx'
```

Does it work locally for you?

### CODE REVIEW

**Comment by mcarans** [comment]
> Your organization is not using the GitHub App Integration. As a result you may experience degraded service beginning May 15th. Please [install the Github App Integration](https://github.com/apps/codecov) for your organization. [Read more](https://about.codecov.io/blog/codecov-is-updating-its-github-integration/?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=OCHA-DAP).

Hmm this is something else to look at!

**Comment by mcarans** [comment]
Having trouble with coveralls showing 0% coverage sigh

**Review by turnerm — APPROVED** [review]
Looks great! We'll need to move AnticiPy over to coveralls too

File: .config/pytest.ini [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [pytest]
-pythonpath = "src"
+pythonpath = ../src
```
[the remark this answers is not in the document]
Comment (mcarans) [reply .config/pytest.ini:2]: For me, `../src` is required for it to work. I think you have probably run `pip install .` which is unnecessary with `pythonpath` set.

It looks like a pytest bug to ignore `rootdir` and use the `pytest.ini` location as the starting point for the `pythonpath`.

[the remark this answers is not in the document]
Comment (turnerm) [reply .config/pytest.ini:2]: Ah interesting, good to know, thanks!

**Pull request merged**